notebooks/data-sources/TestGrid/metrics/blocked_timed_out.ipynb

```diff
@@ -2,67 +2,126 @@
  "cells": [
```

Comment:
<p>what is the relevance of "test_duration" for this calculation? Seems like it could be dropped before saving. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Dropped this value :+1: